src/Dapr.Actors/Runtime/ActorReminder.cs

```diff
@@ -144,5 +172,10 @@ internal ActorReminder(ActorReminderOptions options)
         /// The optional <see cref="TimeSpan"/> that states when the reminder will expire.
         /// </summary>
         public TimeSpan? Ttl { get; }
+        
+        /// <summary>
+        /// Gets the number of invocations of the reminder left.
+        /// </summary>
```

Comment:
Comment should include optional like `Ttl` above.

Reply:
Fixed